app/models/contact.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: contacts
+#
+#  id               :bigint           not null, primary key
+#  name             :string           not null
+#  image            :string
+#  organization     :string
+#  preferred_method :string
+#  personal_number  :string
+#  work_number      :string
+#  email            :string
+#  instagram_url    :string
+#  facebook_url     :string
```

Comment:
For these you could add validations to ensure the values are valid instagram/facebook URLs— similar to what we did in industrial-auth-1

https://github.com/appdev-projects/industrial-auth-1/blob/dc036971554d702962937b67978d22e1b9791a90/app/models/user.rb#L71-L86